zap/src/test/java/org/parosproxy/paros/network/HttpRequestHeaderUnitTest.java

```diff
@@ -204,6 +204,17 @@ void shouldReplaceAnyCookieHeaderWhenSettingCookieParams() {
         assertThat(header.getHeaderValues(HttpHeader.COOKIE), hasSize(1));
     }
 
+    @Test
+    void shouldNotHaveContentLengthHeaderIfGetRequest() throws Exception {
+        // Given
+        HttpRequestHeader header =
+                new HttpRequestHeader("GET / HTTP/1.1\r\nHost: example.com\r\n\r\n");
```

Comment:
Should use the constructor `HttpRequestHeader(String, URI, String)`.

Reply:
done